plugin/src/templates/inject/source-nodes-api-runner.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * This file is used to override node sourcing in Gatsby core so we can add decoupled sourcing to Gatsby
+ * This file was copied from Gatsby core and modified and will be copied back into node_modules/gatsby when Netlify does a cold-cache build.
+ * See 'helpers/files.ts'
+ */
+
```

Comment:
This injected file is copied from the Gatsby repo with a few small changes to work on Netlify